test/functional/device/xctest-e2e-specs.js

```diff
@@ -16,7 +17,7 @@ if (process.env.LAUNCH_WITH_IDB) {
 
     let driver;
 
-    beforeEach(async function () {
+    before(async function () {
```

Comment:
The second test fails due to address already in use error (same as this issue https://github.com/appium/appium-idb/pull/63#discussion_r1049792518) because idb.disconnect() is not called. Is it supposed to be called when session is going to be deleted? If so, I'll fix it later.